apps/src/applab/designElements/CopyElementToScreenButton.jsx

```diff
@@ -13,6 +13,10 @@ const styles = {
   },
   screen: {
   },
+  menu: {
+    maxHeight: '200px',
+    overflowY: 'scroll',
```

Comment:
I'd use `'auto'` but you might have a good reason for `'scroll'`; your call :grin: I think it's an improvement either way!
```suggestion
    overflowY: 'auto',
```